app/string-tester.hoon

```diff
@@ -3,10 +3,10 @@
 !:
 |%
 ++  move  {bone card}
-++  card  $%  $~
+++  card  $%  ~
           ==
 --
-|_  {bowl $~}
+|_  {bowl ~}
```

Comment:
Oops, looks like I didn't 143-ize this tester app. Actually, I think the reason why is because I wanted to remove this entirely and just put the test in a `++test` in the string lib. @mattlevan want to do this and PR @Fang-'s [string library repo](https://github.com/Fang-/urbit-string) when it's tested and done?